### ISSUE TITLE
Moderators for w2ui.com Disqus needed

### ISSUE THREAD

**Issue by mrg2001** [vitmalina/w2ui#1546]
If you have a bit time you can donate to the cause, it will be much appreciated. Usually, questions are quite simple, but it greatly helps people get going. I would like to reserve issue on GitHub for bugs/feature requests only and all other questions to w2ui.com disqus comments

Drop a line into this thread and post you disqus login/email and I will make you a moderator.

**Comment by aadrian** [comment]
@mrg2001 
IMHO Disqus is really not the right tool for this kind of support (hence why so many have abandoned it and use something else instead) . It's main usage is "general comments for blogs".

**Comment by mpf82** [comment]
I agree with @aadrian, disqus seems like a bad choice as a support platform for w2ui. 

What's wrong with pointing users with general questions to stackoverflow (or short "*SO*")? The advantage is that it's an established place  to look for help and there's a vast user base, that can not only help with w2ui questions, but other JS/PHP/Python/... questions as well. SO is easy to use, easy to search and I guess it's more likely that a developer already has a SO account than a disqus account.

Just my 2 cents =)

**Comment by aadrian** [comment]
IMO there are a few things that could be done to make life around W2UI easier for everybody:
 
 - move this project to a GitHub Organization https://github.com/blog/674-introducing-organizations  and https://help.github.com/articles/creating-a-new-organization-account/   to be able to easier manage users that could help, and avoid relying on only a few users to merge pull requests and publish updates.

Than we could start helping w2ui much easier:
 - For simple chat one could use **Gitter** https://gitter.im (since it integrates with GitHub nicely, and so many projects are already using it) and mention this on the main page, together with the SO links.
 - use *"documentation"*,  *"support"*,  *"v1.4"* and *"v1.5"* labels to back-label all issues, so that existing solutions ca be easier found.
 - use https://github.com/blog/2111-issue-and-pull-request-templates to make sure future issues comply.

 Other stuff easier to achieve by the community that helps:
 - collect some still working solutions from closed issues and Disqus threads and add them to the docs and a FAQ page.
 - generate a PDF from the docs too (and link it to a printing service as Lulu)
 - convert the docs part of the site from a PHP one to a GitHub site, since hosting is free and can be easier deployed.
 - do a few surveys (easy to make) and ask the users what are they missing (since many of them won't just create issues if they miss something)
 - a memorable logo - most JS Frameworks have one :) .

Since there doesn't seem to exist books and video courses about W2UI yet, most new users still need to painfully check all the above sources anyway, so IMHO consolidating the documentation could greatly help.
